src/openrct2/scripting/bindings/game/ScContext.hpp

```diff
@@ -232,6 +232,22 @@ namespace OpenRCT2::Scripting
             }
         }
 
+        std::vector<DukValue> getAllTrackSegments()
```

Comment:
I did this instead of exposing a property to context with the number of track elements. I could still add that too.